port/raspberrypi/rp2xxx/src/hal/irq.zig

```diff
@@ -139,3 +169,48 @@ pub fn disable(mask: Mask) void {
         },
     }
 }
+
+const VtorFunction = *const fn () callconv(.C) void;
+
+const nvic_interrupt_count = switch (chip) {
+    .RP2040 => 25,
+    .RP2350 => 51,
+};
+
+pub var ram_vtor: [16 + nvic_interrupt_count]VtorFunction align(256) = undefined;
```

Comment:
Maybe we should let the linker decide where to place this.

With #381 we'd want to move the stack to before .data and .bss. In that case I think we'd probably want it right as the end of RAM (making it easy to peek at)

Reply:
I had thought about that and didn't see much benefit.   One thing to consider is that the vector table needs to be on a 256 byte boundry.  I don't know if Zig is smart enough to put other variables in any space needed for padding; but if Zig is smart enough, putting the vectors in a separate section might be more wasteful.

Reply:
Because it would waste space between the end of the table and the end of flash (if we put it add flash_end - 256)?

Reply:
Perhaps it's silly for me to worry about wasting 100 bytes on a chip with 264K.  I started my programming journey on machines like the PDP-8 (where you only had 4K) and developed multi-user interactive systems for the PDP-11 (a whopping 64K), so every byte was precious.

Reply:
I don't blame you. I was actually just curious.